tests/Behat/Context/Application/EmailsContext.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Application;
+
+use Behat\Behat\Context\Context;
+use Tests\Sylius\RefundPlugin\Behat\Services\Provider\MessagesProvider;
+use Webmozart\Assert\Assert;
+
+final class EmailsContext implements Context
+{
+    /** @var MessagesProvider */
+    private $messagesProvider;
+
+    public function __construct(MessagesProvider $messagesProvider)
+    {
+        $this->messagesProvider = $messagesProvider;
+    }
+
+    /**
+     * @Then an email with credit memo should be sent again to :email
+     */
+    public function quantityOfEmailsForCustomerWithCreditMemo(string $email): void
+    {
+        Assert::eq($this->countMessagesToCustomer($email), 2);
+        Assert::true($this->lastEmailBodyContains('Some of the units from your order have been refunded.'));
+    }
+
+    private function countMessagesToCustomer(string $email): int
+    {
+        $counter = 0;
+        /** @var \Swift_Message $message */
+        foreach ($this->messagesProvider->getMessages() as $message) {
+            if (array_key_exists($email, $message->getTo()) && false !== strpos($message->getBody(), 'Some of the units from your order have been refunded.')) {
```

Comment:
`'Some of the units from your order have been refunded.'` I think it should be a parameter